src/main/java/org/jenkinsci/plugins/github_branch_source/AbstractGitHubSCMSource.java

```diff
@@ -49,11 +49,7 @@
 import jenkins.scm.api.SCMSourceCriteria;
 import jenkins.scm.api.SCMSourceDescriptor;
 import jenkins.scm.api.SCMSourceOwner;
-import org.kohsuke.github.GHMyself;
-import org.kohsuke.github.GHOrganization;
-import org.kohsuke.github.GHRepository;
-import org.kohsuke.github.GHUser;
-import org.kohsuke.github.GitHub;
+import org.kohsuke.github.*;
```

Comment:
Please read @stephenc contributing guideline + checkstyle rules, they both against `*`. Every time you adding/removing classes imports will be collapsed to `*` or expanded to single that provides big useless change. i.e. here it is not obvious what was added. 
PS in general there is no difference in performance because this imports influence only on build time AFAIK